enricher/standard/src/test/java/io/fabric8/maven/enricher/standard/DefaultServiceEnricherTest.java

```diff
@@ -184,6 +186,7 @@ private String enrich() throws com.fasterxml.jackson.core.JsonProcessingExceptio
         KubernetesList list = builder.build();
         assertEquals(list.getItems().size(),1);
 
+        System.out.println(list);
```

Comment:
can we avoid this ?

Reply:
Aaah , sorry i messed up the branch. I'll revert the latest unfinished commit.